web/src/components/network/AddressesDataList.tsx

```diff
@@ -39,15 +39,23 @@ import {
 } from "@patternfly/react-core";
 
 import { FormLabel } from "~/components/core";
-import { IpAddressInput, IpPrefixInput } from "~/components/network";
+import IpAddressInput from "~/components/network/IpAddressInput";
+import IpPrefixInput from "~/components/network/IpPrefixInput";
 import { _ } from "~/i18n";
+import { IPAddress } from "~/types/network";
 
 let index = 0;
 
+type Address = IPAddress & { id?: number };
+
 export default function AddressesDataList({
   addresses: originalAddresses,
   updateAddresses,
   allowEmpty = true,
+}: {
+  addresses: Address[];
+  updateAddresses: (addresses: Address[]) => void;
+  allowEmpty: boolean;
```

Comment:
I am not much familiar with ts so just curious. When `allowEmpty` is predefined as true. shouldn't here be used `allowEmpty?` to indicate that it is optional and will be replaced by true? or it applies only if it is checked later for presence and uses true dynamically?

Reply:
> I am not much familiar with ts so just curious. When `allowEmpty` is predefined as true. shouldn't here be used `allowEmpty?` to indicate that it is optional and will be replaced by true?

Good catch. I'll check it tomorrow when on front of laptop and editor, but lost probably you're right